AssertionError on test_logging.py solved by strip()

I have an assertion error on `logs == expected, logs` on the last line of test_logging.py when running pytest.
I think it is simply an issue with trailing spaces as replacing the end of the file by
```python
logs = logs.strip()
expected = expected.strip()
assert logs == expected, logs
```
yields a test success.

The exact assertion error is :
```
E       AssertionError:            INFO     Read configuration files:
                                                                                                                                     config.py:
E                    INFO
                                                                                                           scene_file_writer.py:
E                             File ready at

E
E       assert '           I...           \n' == '           I...           \n'
E         Skipping 760 identical leading characters in diff, use -v to show
E         Strings contain only whitespace, escaping them using repr()
E         - '          \n

           \n'
E         + '          \n'

tests/test_logging/test_logging.py:58: AssertionError

```
I am running `rich` on version 3.3.2.

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,6 +7,6 @@ pycairo
 pydub
 pygments
 pyreadline; sys_platform == 'win32'
-rich
+rich>=4.2.1
 # pytest (for development only)
 # black (for development only)

